clients/rancher/generated/management/v3/zz_generated_cluster_role_template_binding_status.go

```diff
@@ -0,0 +1,24 @@
+package client
```

Comment:
I am not sure why this file was generated when I ran `go generate`, but I have added it for now.

Reply:
Discussed this one offline